dcmdata/include/dcmtk/dcmdata/dclist.h

```diff
@@ -101,13 +61,13 @@ class DCMTK_DCMDATA_EXPORT DcmList
     /// destructor
     ~DcmList();
 
-    /** insert object at end of list
+    /** insert object at end of list and make it be the "current" one
```

Comment:
Document the existing behavior for clarity (might be relevant on the call site).